src/main/java/graphql/nadel/FieldInfo.java

```diff
@@ -5,9 +5,7 @@
 public class FieldInfo {
 
     public enum FieldKind {
```

Comment:
I am not sure if we need field info at all, we can change parser such that FieldDefinition will contain service name(e.g. NadelFieldDefinition extends FieldDefinition). For example, for hydration it is not needed at all cause I can get this information from FieldDefinitionWithTransformation (see SourceQueryTransformer)